api/README.md

```diff
@@ -105,21 +105,21 @@ Generate a geotiff for any wfp raster using the stac API and saves it in S3. It
 To run the api locally, run:
 
 ```
-make api
+source set_envs.sh && make api
```

Comment:
I don't think we want to rely on AWS for everything. Not all the developers working on PRISM have access to the AWS console.

Reply:
Makes sense.  It's not clear to me what should be able to be tested at what stage of development, given that we're sometimes passing in inactive API keys.